packages/cxl-ui/src/components/cxl-popover-card.js

```diff
@@ -0,0 +1,193 @@
+import { html, LitElement, render } from 'lit';
+import { customElement, property, query } from 'lit/decorators';
+import '@conversionxl/cxl-lumo-styles';
+import cxlPopoverCardStyles from '../styles/cxl-popover-card-css';
```

Comment:
PhpStorm shows

![Screenshot 2021-09-01 at 16 16 20](https://user-images.githubusercontent.com/5165721/131687572-84c33668-14cb-4bff-9006-fa89bd8fe122.png)

Reply:
@pawelkmpt styles are not built probably (file should exist under this path). Try `yarn build`.